utils/build-script-impl

```diff
@@ -640,6 +642,18 @@ function set_build_options_for_host() {
         -DLLVM_BUILD_EXTERNAL_COMPILER_RT:BOOL="$(false_true ${SKIP_BUILD_COMPILER_RT})"
     )
 
+    if [[ "${SWIFT_TOOLS_NUM_PARALLEL_LINK_JOBS}" != "" ]] && [[ $(is_swift_lto_enabled) == "FALSE" ]]; then
+        swift_cmake_options+=(
+            "-DSWIFT_PARALLEL_LINK_JOBS=${SWIFT_TOOLS_NUM_PARALLEL_LINK_JOBS}"
```

Comment:
I haven't looked at this, but I wonder if it would make sense to just have one option for this rather than have two options. Is that possible? How do we do it in the lldb case?